cypress/support/fragments/finance/fiscalYears/fiscalYears.js

```diff
@@ -74,5 +86,23 @@ export default {
       deleteButton.click(),
       Button('Delete', { id:'clickable-fiscal-year-remove-confirmation-confirm' }).click()
     ]);
-  }
+  },
+
+  createViaApi: (fiscalYearProperties) => {
+    return cy
+      .okapiRequest({
+        path: 'finance/fiscal-years',
+        body: fiscalYearProperties,
+        method: 'POST'
```

Comment:
Please add a parameter to the okapiRequest:     isDefaultSearchParamsRequired: false,

Reply:
Done